torch/csrc/inductor/aoti_runner/model_container_runner.cpp

```diff
@@ -34,8 +34,22 @@ AOTIModelContainerRunner::AOTIModelContainerRunner(
   get_call_spec_func_ = reinterpret_cast<decltype(get_call_spec_func_)>(
       model_so_->sym("AOTInductorModelContainerGetCallSpec"));
 
-  AOTI_RUNTIME_ERROR_CODE_CHECK(
-      create_func_(&container_handle_, num_models, is_cpu, cubin_dir));
+  AOTI_RUNTIME_ERROR_CODE_CHECK(create_func_(
+      &container_handle_,
+      num_models,
+      is_cpu,
+      cubin_dir.empty() ? nullptr : cubin_dir.c_str()));
+
+  /*
+  #ifdef FBCODE_CAFFE2
+    std::string extern_kernel_nodes_path =
+        model_so_path.substr(0, model_so_path.size() - 3) + ".json";
+    if (fileExists(extern_kernel_nodes_path)) {
+      proxy_executor_handle_ =
+          std::make_unique<torch::aot_inductor::FbProxyExecutor>(
+              extern_kernel_nodes_path, is_cpu);
+  #endif
+  */
 }
```

Comment:
Why the commented out code?

Reply:
I was trying to keep this as a reminder to add initialization for `proxy_executor_handle_` (partially answering your later question about it). Unfortunately, there is no equivalent of `FbProxyExecutor` in OSS yet. I will add a TODO at the declaration point of `proxy_executor_handle_` and remove the comments here.